### DISCUSSION TITLE
Our Privacy Opportunity course, Lesson 2, Concept 2 there is a sound break

### DISCUSSION THREAD

**Discussion by ghost** [OpenMined/courses#148] [Feedback] [answered]
There is a small sound break at 3:07 in the video. It is really not that big of an issue, just want to be helpful in making this course perfect :D

**Comment by yemikifouly** [comment]
Hey Tofnaz! Don't worry. Keep the feedback coming. We want those. 😀 

Alright! Onto your feedback. Just so I am clear did you mean the concept titled "What is an Information Flow?"

**Reply by ghost** [reply]
Yes, that one

**Comment by em-blue** [comment, marked as answer]
Hey @tofnaz we've gone back over the video files, but unfortunately I think there was a hardware error with the recording! The word missing is "for". We might go back and insert Andrew saying "for" at some point to fix it haha

**Reply by ghost** [reply]
Ahahah, anyway it is a very minor and insignificant issue. Definitely doesn't affect the overall quality of this great course!
